tests/testrom.c

```diff
@@ -295,6 +298,7 @@ int main() {
 
 	int64_t total_time = TIMER_MICROS(stop-start) / 1000000;
 
-	printf("\nTestsuite finished in %02lld:%02lld\n", total_time%60, total_time/60);
+	console_set_debug(true);
```

Comment:
IIRC this will not work on an ed64 without something listening on the USB. I'm not sure of the cause b/c I know it is not a technical requirement. Let's merge this and I will try to fix it later.

Reply:
It's a leftover of the debugging library PR. The ed64 backend in usb.c doesn't have a timeout. It might require a modification in `usb_everdrive_wait_pidma` or `usb_everdrive_usbbusy` or both, to add a simple counter-based timeout like in `usb_64drive_waitidle`. If you could do this it would be great.